src/ua/hillel/FindSymbolOccurance.java

```diff
@@ -0,0 +1,18 @@
+package ua.hillel;
+
+public class FindSymbolOccurance {
+    public static void main(String[] args) {
+        String line = "You are your only limit";
```

Comment:
Please extract all code from method main to a separate method with the name `findSymbolOccurrance()`